src/client/react.js

```diff
@@ -160,7 +162,24 @@ export async function signOut(options = {}) {
 
 /** @param {import("types/react-client").SessionProviderProps} props */
 export function SessionProvider(props) {
-  const { children, baseUrl, basePath, staleTime = 0 } = props
+  const {
+    children,
+    baseUrl,
+    basePath,
+    staleTime = 0,
+    refetchOnWindowFocus = true,
+    refetchInterval,
+  } = props
+
+  let broadcastSession
+  let broadcastSignOut
+  if (typeof props.broadcast === "boolean") {
```

Comment:
I would not support both `boolean` and `{}`, I think it's ambiguous and confusing, if people want to disable all events they can do it manually, they are just 2 events and I don't think this number will grow much further 😄 
```js
broadcast: {
  session: false,
  signout: false
}
```

You are destructuring all the `props` above but not this one? Perhaps to keep consistency I would destructure also `broadcast` on line 172 😄

Reply:
I agree with @mtt87; it is good to keep consistency on the type the prop expects. 

For now, if we only have two options is not necessary, but in the future, we could enable an extra switch to disable all at once:
```js
broadcast: { 
  disabled: true
}

```